src/styles/_catalog.scss

```
@@ -1,22 +1,26 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  @extend %grid-center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  grid-template-columns: repeat(1, $card-width);
```

```suggestion
  grid-template-columns: $card-width;
```

Unnecessary function call.